system/jlib/jlib.hpp

```diff
@@ -95,7 +95,7 @@ class CIArrayOf : public CIArray
     inline TYPE & tos(aindex_t num) const         { return (TYPE &)CIArray::tos(num); }
     inline TYPE **getArray(aindex_t pos = 0)      { return (TYPE **)CIArray::getArray(pos); }
     inline void append(TYPE& obj)                 { assert(&obj); CIArray::append(obj); } 
-    inline void appendUniq(TYPE& obj)             { assert(&obj); CIArray::appendUniq(obj); } 
+    inline bool appendUniq(TYPE& obj)             { assert(&obj); return CIArray::appendUniq(obj); }
```

Comment:
Not sure why these didn't return bool? I can pull these out to a separate Jira/PR if wanted.